components/onboarding_explore_tools_tour/explore_tools_tour_tip.scss

```diff
@@ -0,0 +1,3 @@
+.tour-tip__pulsating-dot-ctr {
```

Comment:
This will override the [other style declaration](https://github.com/mattermost/mattermost-webapp/blob/master/components/widgets/tour_tip/tour_tip.scss#L37) for other usages. Sass compilation loads all the style rules at the same time, and doesn't do any automatic class name changing like styled-components, so we need to scope this on our own.

If we're sure we need this for the explore tools tour tip case, can we either add another class to the element with this class, or limit it its scope by adding a specific class to some parent element, and making this apply only to pulsating dots within that parent?